README.md

```diff
@@ -82,6 +82,16 @@ Also, additional utils from `sources/utils/` are used in tests and deployment sc
 
 `tact.config.json` contains the configuration for the Tact compiler. In most cases, you don't need to modify this file.
 
+## Smart-Contracts Structure
+
+Main smart contract is `jetton_minter.tact`. It imports `messages.tact` and `jetton_wallet.tact`, so they will be compiled automatically, when setting `jetton_minter.tact` as target in `tact.config.json`.
```

Comment:
Perhaps it would be better to include the Mermaid diagrams for project imports and trait inheritance from the compilation report (GitHub renders those nicely)